PADC_FP_Travelling/app/src/main/res/layout/activity_attraction_detail.xml

```diff
@@ -0,0 +1,148 @@
+                android:tint="@color/primary_text"
+                android:layout_below="@id/iv_category"
+                android:layout_marginTop="8dp"
+                />
+
+            <TextView
+                android:id="@+id/tv_location"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="Yangon"
+                android:layout_below="@id/tv_category"
+                android:layout_marginLeft="32dp"
+                android:textSize="@dimen/text_regular_2x"
+                android:fontFamily="sans-serif-light"
+                android:layout_marginTop="8dp"
+                />
+
+            <Button
+                android:id="@+id/btn_map"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="Search in Map"
+                android:layout_below="@id/tv_location"
+                android:layout_marginLeft="32dp"
+                />
+
+            <TextView
+                android:id="@+id/tv_attraction_desc"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/margin_small"
+                android:fontFamily="sans-serif-light"
+                android:lineSpacingMultiplier="1.2"
+                android:text="Mandalay was the last Royal Capital of Myanmar and is located nearly 700 km north of Yangon between the mighty Ayeyarwaddy River and the Shan plateau. Mandalay is considered the centre of Burmese culture, a city of artisans and a trading centre of goods in all directions. The geometric laid out streets, the many bicycles riders, the 8 km long moth and wall around the former Royal Palace, the pagodas and monasteries abound and the sacred Mandalay Hill towering over it all, are some of its most visible features. In the vicinity of Mandalay are the former royal capitals and religious centers of Amarapura, Mingun, Inwa (Ava) and Sagaing, each worth visiting. Mandalay is another gateway to Myanmar and has direct air links with Bangkok, Singapore, Kunming and Chiang Mai.\n\n Mandalay was the last Royal Capital of Myanmar and is located nearly 700 km north of Yangon between the mighty Ayeyarwaddy River and the Shan plateau. Mandalay is considered the centre of Burmese culture, a city of artisans and a trading centre of goods in all directions. The geometric laid out streets, the many bicycles riders, the 8 km long moth and wall around the former Royal Palace, the pagodas and monasteries abound and the sacred Mandalay Hill towering over it all, are some of its most visible features. In the vicinity of Mandalay are the former royal capitals and religious centers of Amarapura, Mingun, Inwa (Ava) and Sagaing, each worth visiting. Mandalay is another gateway to Myanmar and has direct air links with Bangkok, Singapore, Kunming and Chiang Mai."
```

Comment:
Use tools:text to show in preview mode only. Also put the string as separate resource in strings.xml